.travis.yml

```diff
@@ -5,7 +5,7 @@ services:
   - mongodb
   - docker
 node_js:
-  - "0.10"
+  - "6.9"
```

Comment:
Sorry I should have said `6.9.1`, just to keep it consistent across all templates